packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -549,7 +559,17 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
                 _stampAncillaryData(ancillaryData, requester)
             );
             bool disputeSuccess = request.resolvedPrice != request.proposedPrice;
-            payout = request.bond.mul(2).add(request.finalFee).add(request.reward);
+            uint256 bond = request.bond;
+
+            // Unburned portion of the loser's bond = floor(bond / 2)
+            uint256 unburnedBond = bond.div(2);
```

Comment:
Is there any way to link `unburnedBond` and `burnedBond` via state variables? It's not very readable how `burnedBond + unburnedBond = bond`.

However, I don't really like the idea of storing `burnedBond` on-chain because it contributes bytecode and gas to store a variable that isn't that important to the user, so I understand why you did it this way.

Reply:
Maybe we could use a helper method to standardize the computation?

Reply:
Yeah, we could return `burnedBond` + `unburnedBond` + an assertion that their sum equals `bond` which should help reduce some of the testing code

Reply:
I added a function to compute the burned bond and compute unburned bond by doing bond - burned bond, which should guarantee burned bond + unburned bond = bond.